js/views/app_view.js

```diff
@@ -196,5 +196,15 @@
       const dialog = new Whisper.SeedDialogView({ seed });
       this.el.append(dialog.el);
     },
+    showDevicePairingDialog() {
+      const dialog = new Whisper.DevicePairingDialogView();
+      Whisper.events.on('devicePairingRequestReceived', pubKey =>
```

Comment:
Do we need to *unregister* these events after the dialog is closed? And would we be duplicating events if the dialog is opened multiple times?

Reply:
Yes it would create multiple events. We should removeListener when dialog is closed or before we add the listener.